nevergrad/optimization/differentialevolution.py

```diff
@@ -7,6 +7,8 @@
 import numpy as np
 import nevergrad.common.typing as tp
 from nevergrad.parametrization import parameter as p
+from .metamodel import MetaModelFailure as MetaModelFailure
+from .metamodel import _learn_on_k_best as _learn_on_k_best
```

Comment:
```suggestion
from . import metamodel
```
Then just use metamodel._learn_k_best etc...
Then again, that means it should probably be removed to remove the initial _

the syntax import X as X is only necessary for external interfaces, so not here